scripts/ci/build/build-app.sh

```
@@ -53,12 +53,17 @@ done
 
 shift $((OPTIND - 1))
 
-if [ -z "$commit" ]; then
-  commit=$(git log -n 1 --pretty=format:%H -- app/client/mobile | cut -c1-8)
-fi
-
-file_name="build-$ENV-$commit.apk"
+if [ -z "$tag" ]; then
+  if [ $ENV = "dev" && -z $PR_NUMBER ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)